interfaces/builtin/bluez.go

```diff
@@ -91,7 +96,7 @@ var bluezConnectedPlugAppArmor = []byte(`
 # Allow all access to bluez service
 dbus (receive, send)
     bus=system
-    peer=(label=bluez5_bluez_*),
+    peer=(label=snap.bluez.*),
```

Comment:
This was my understanding as well. With this fix we can have a working bluez snap in the store today (as opposed to the broken one that's in there now) while we wait on the proper fix.